miden-tx/src/verifier/mod.rs

```diff
@@ -58,12 +58,13 @@ impl TransactionVerifier {
 
     // HELPERS
     // --------------------------------------------------------------------------------------------
+
     /// Returns the consumed notes commitment.
-    fn compute_consumed_notes_hash(consumed_notes: &[ConsumedNoteInfo]) -> Digest {
+    fn compute_consumed_notes_hash(consumed_notes: &[Nullifier]) -> Digest {
```

Comment:
Should you rename this variable `consumed_notes` if it only contains an array of `Nullifiers`?

Reply:
Possibly - but I'll tackle this in a different PR when I refactor the structure to use input/output note terminology.